buildable/src/main/java/buildable/config/BuildField.java

```diff
@@ -0,0 +1,19 @@
+package buildable.config;
+
+
+import buildable.annotation.BuiltWith;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Documented
+@Retention(RetentionPolicy.SOURCE)
+public @interface BuildField {
```

Comment:
The other annotations have pretty verbose javadoc, which I quite like (really the only kind of javadoc that's worth doing, IMO), we should be consistent.